package.json

```diff
@@ -15,7 +15,7 @@
     "type-check": "tsc --project tsconfig.validation.json",
     "type-check:cypress": "tsc --project cypress/tsconfig.json",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx --max-warnings 0 --report-unused-disable-directives",
-    "lint:fix": "pnpm lint -- --fix",
```

Comment:
The standalone `--` was necessary for npm, but it's not for pnpm.

Reply:
Cool! We can drop the confusing double `--`! 🥳